service/controller/legacy/v27/resource/ipam/create.go

```diff
@@ -275,17 +275,22 @@ func statusClusterNetworkCIDR(cluster cmav1alpha1.Cluster) string {
 }
 
 func mustG8sClusterStatusFromCMAClusterStatus(cmaStatus *runtime.RawExtension) g8sv1alpha1.AWSClusterStatus {
-	if cmaStatus == nil {
```

Comment:
We have some backported functionality in the legacy controller that has to be aligned as well.